### ISSUE TITLE
Review node reading

### ISSUE THREAD

**Issue by Irineu333** [Irineu333/SpeakTouch#40]
To revise and refactor the node reading system.

Note: Use Talkback's behavior as a parameter, since it is also a reference for developers.

**Comment by Irineu333** [comment]
## Result:

### Correct:

- Reading containers works.

### Wrong:

- It is not reading the isSelected.

**Comment by Irineu333** [comment]
I ran into a problem trying to read `isSelected`. There is the possibility that a container, representing only one item on the screen, has the state `isSelected` both in the parent and in the child. In this case the scanner will read the "selected" status twice. Talkback doesn't have the same problem.

### Speak Touch
https://user-images.githubusercontent.com/45833588/234166490-b965142f-29ad-4bae-aca5-375d239f3278.mp4

### Takback 
https://user-images.githubusercontent.com/45833588/234166513-b3e318fa-6bc6-479d-ab95-b6bdf0e64095.mp4

**Comment by Irineu333** [comment]
The talkback reads the isSelected from the container only. I will adopt the same solution.

**Comment by Irineu333** [comment]
Several improvements made to the node reader, in order to get closer to talkback. I believe it is the safest since talkback is the reference, in the future, when there are more users and a better understanding, we can diverge more.

**Comment by Irineu333** [comment]
More points to review:
- [ ] Custom EditField
- [ ] Custom Checkable
